$NetBSD: patch-.._spine.1,v 1.2 2012/12/11 18:18:59 ahp-nils Exp $

Replace the copyright year and version number to match the package
--- ../spine.1.orig	2012-12-11 17:35:04.000000000 +0000
+++ ../spine.1
@@ -1,12 +1,12 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.36.
-.TH SPINE "1" "March 2009" "SPINE 0.8.7c  Copyright 2002-2008 by The Cacti Group" "User Commands"
+.TH SPINE "1" "March 2009" "SPINE @VERSION@  Copyright 2002-2012 by The Cacti Group" "User Commands"
 .SH NAME
 SPINE \- High-speed polling agent for cacti
 .SH SYNOPSIS
 .B spine
 [\fIoptions\fR] [\fIfirstid lastid\fR]
 .SH DESCRIPTION
-SPINE 0.8.7c  Copyright 2002\-2008 by The Cacti Group
+SPINE @VERSION@  Copyright 2002\-2012 by The Cacti Group
 .SH OPTIONS
 
 .TP
